docs/content/getting-started/setup-openfga.mdx

```diff
@@ -27,43 +27,24 @@ This will start an HTTP server and gRPC server with the default configuration op
 
 ## Configuring The Server
 
-There are several configuration options for OpenFGA. One option is to pass them in with an environment file. For example:
+You may configure the OpenFGA sever with `config.yaml` file. The config.yaml is be specified in either 
+- `/etc/openfga`
+- `$HOME/.openfga`
+- `.` (i.e., the current working directory).
 
-```
-$ cat ./myenv
-OPENFGA_DATASTORE_ENGINE=postgres
-OPENFGA_AUTH_PRESHARED_KEYS=key1,key2
-# etc
+The OpenFGA will search for the configuration file in the above order.
+
+An example of configuration to use postgres as database and configured with preshared authentication keys "key1, key2" is:
```

Comment:
More succintly,

"Here is a sample configuration that leverages postgres and preshared authentication:"

Reply:
Done